### PR TITLE
[NOT A JOKE] Aether Jar

### PR DESCRIPTION
**Pull request by jjpark-kb**
## About The Pull Request
Aether has pulled the short straw and has made me very upset; Aether needs a fitting punishment.

## How This Contributes To The Skyrat Roleplay Experience
Comedy.

## Changelog

:cl:
add: adds aether's jar
/:cl:

### CODE REVIEW

**Review by SarmentiCampbell — COMMENTED** [review]
look good to me, speed merge

**Comment by Paxilmaniac** [comment]
the skyrat codebase was a mistake

File: modular_skyrat/master_files/code/game/objects/items/aether_jar.dm [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+/obj/item/aether_jar/Initialize(mapload)
+	. = ..()
+	START_PROCESSING(SSobj, src)
+
+/obj/item/aether_jar/Destroy(force)
+	STOP_PROCESSING(SSobj, src)
+	. = ..()
+
+/obj/item/aether_jar/process(delta_time)
+	for(var/mob/moving_mob in GLOB.clients)
```
Comment (tf-4) [inline modular_skyrat/master_files/code/game/objects/items/aether_jar.dm]:
```suggestion
	for(var/mob/moving_mob as anything in GLOB.player_list)
```
There are only clients in `GLOB.clients`

**Comment by jjpark-kb** [comment]
smh

**Pull request closed without merging**